README.md

```diff
@@ -111,12 +107,11 @@ If you need to create a new Slack command:
  1. Go to the Slack Commands config page for your team's Slack app: `https://api.slack.com/apps/<YourSlackAppID>/slash-commands`
  2. Click on "Create New Command"
    * Fill in the slash command (e.g. `/foo`)
-   * Enter `https://<appname>.herokuapp.com/<command>` as the request URL, replacing `<appname>` with the name of your Heroku app instance (e.g. `stevenson-bot`) and `<command>` by the command name (e.g. `foo`)
+   * Enter the URL this command will trigger – e.g. if your instance will be hosted on Heroku  it will look like `https://<appname>.herokuapp.com/<command>`, replacing `<appname>` with the name of your Heroku app instance (e.g. `stevenson-bot`) and `<command>` by the command name (e.g. `foo`)
```

Comment:
```suggestion
   * Enter the URL this command will trigger – e.g. if your instance will be hosted on Heroku it will look like `https://<appname>.herokuapp.com/<command>`, replacing `<appname>` with the name of your Heroku app instance (e.g. `stevenson-bot`) and `<command>` by the command name (e.g. `foo`)
```